packages/amplify-codegen-appsync-model-plugin/src/configs/dart-config.ts

```diff
@@ -0,0 +1,17 @@
+export const LOADER_CLASS_NAME = 'ModelProvider';
+export const BASE_IMPORT_PACKAGES = [
+  'package:flutter/foundation.dart',
+  'package:amplify_datastore_plugin_interface/amplify_datastore_plugin_interface.dart'
+];
+export const COLLECTION_PACKAGE = 'package:collection/collection.dart';
+
+export const typeToEnumMap: { [name: string] : string} = {
+  String: 'string',
+  Int: 'int',
+  Float: 'double',
+  Boolean: 'bool',
+  AWSDate: 'date',
+  AWSDateTime: 'dateTime',
+  AWSTime: 'time',
+  AWSTimestamp: 'timestamp',
```

Comment:
A few AWS* types missing here, is that intentional? Also the mapped values are different then in scalar map, why?

Reply:
This is not the mapping for graphql type to dart native type but to a enum type designed by flutter team. This is not finalized since flutter team is still discussing with ios and android team. I will update this after they have a final result.

Reply:
Update: add the missing AWS types